### ISSUE TITLE
Click and Hover system events must capture DOM obj & object title

### ISSUE THREAD

**Issue by NejcZdovc** [blotoutio/sdk-js#7]
SDK should be able to capture DOM obj - complete path when nested.

Object Title use cases:
SDK should be able to capture DOM object title as :
- text displayed to the users.
- ALT text from images when image.
- when image without ALT text or object without title then ID
- When image without ALT text but with link then link
- As last resort DOM - object identifier or standard default as per DOM.

Current Behaviour:
- Right now we when I hover over images then though link or ids are present but title is missed because there is no ALT text present & link or IDs are not sent.

Also ALT text from images should be default choice for title if present after link.

----

Slack icon I was hovering when this event was sent - but I can't know.

Also let's set as recommendation in our documentation that ALT should be present for better analysis then link if ALT is not needed.

![image](https://user-images.githubusercontent.com/9574457/111426165-84051580-86f4-11eb-9eb0-814b9fafdda4.png)
